content/uc-doc/upgrade/upgrade_notes.md

```diff
@@ -2,6 +2,18 @@
 title: Upgrade notes
 ---
 
+## 23.14 {#23-14}
+
+- `wazo-dird`: when creating and editing a source for the `phonebook`
+  backend(`/backends/phonebook/sources`), the `phonebook_uuid` value is now validated to point to an
+  existing `phonebook`(available from the `/phonebooks` API from the same tenant)
+- `wazo-dird`: deleting a phonebook from the `/phonebooks` API now also automatically deletes any
+  phonebook sources(`/backends/phonebook/sources`) pointing to that phonebook
+- `wazo-dird`: database migrations ensures that all existing phonebook sources now include a
+  `phonebook_uuid` pointing to a valid phonebook. Sources which cannot be resolved to an existing
+  phonebook are deleted(a backup of the database and wazo system prior to upgrading to this version
+  is strongly recommended).
+
```

Comment:
Upgrade notes are more "Breaking changes"
None of these point are relevant in this section. The last point was already mentioned in 23.11

Reply:
IMHO we need to say that sources will be deleted in 23.13. "A future release" is not precise enough.